README.md

```diff
@@ -1,6 +1,4 @@
-# The Go SDK for the Blocky Attestation Service Wasm Runtime
-
-(working name)
+# Go SDK for the Blocky Attestation Service Wasm Runtime (BASM)
```

Comment:
Figured the "working name" has stuck enough for now that we can remove this note.